src/page/order-summary.page.ts

```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class OrderSummaryPage {
+  private finalButton: ElementFinder;
+
+  constructor () {
+    this.finalButton = $('#center_column > div > p > strong');
+  }
+
+  public async summaryOrder(text: string): Promise<void> {
+    await expect(this.finalButton.getText()).toBe(text);
```

Comment:
you should not add assertions into a PO

Reply:
¿Expect? then I should put in buy-tshirt example: await expect( orderSummaryPage.summaryOrder() ....etc ?